.github/workflows/ci.yml

```diff
@@ -97,7 +97,6 @@ jobs:
         run: |
           MODIN_ENGINE=dask python -c "import modin.pandas as pd; print(pd.DataFrame([1,2,3]))"
           MODIN_ENGINE=ray python -c "import modin.pandas as pd; print(pd.DataFrame([1,2,3]))"
-          MODIN_ENGINE=unidist UNIDIST_BACKEND=mpi mpiexec -n 1 python -c "import modin.pandas as pd; print(pd.DataFrame([1,2,3]))"
```

Comment:
We still need to test the clean install of unidist using pip.

Reply:
Added for ubuntu.

Reply:
Why also not for Windows?

Reply:
There is a little bit more stuff to install for MPI on Windows. I don't even know if it is possible to install all that stuff without interacting with GUI. Originally, Modin tests for unidist on MPI were added only on Linux so I think it is okay to leave the things here on Linux too.

Reply:
Ok, just curious: maybe it’s enough to just get the prerequisites for MPI through conda?

Reply:
It should be enough but in that case we would need to switch to `conda-incubator/setup-miniconda@v2` usage here, which I didn't want to.